browser/notifications/brave_alert_dispatcher_mac.mm

```diff
@@ -48,6 +52,26 @@ - (void)closeNotificationWithId:(NSString *)notificationId
   }
 }
 
+- (void)closeNotificationsWithProfileId:(NSString*)profileId
```

Comment:
is this code copied from upstream?

Reply:
Not directly copied: adapted from https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/notifications/stub_alert_dispatcher_mac.mm. Modified to get `* notificationCenter` and notifications from there - similarly to what `closeNotificationWithId` method above does.